app/views/categories/index.html.erb

```diff
@@ -0,0 +1,49 @@
+<section class='categories_controllers'>
+     <header class="input-container">
+      <nav>
+        <ul class='links'>
+          <li>
+            <i class="fa fa-bars"></i>
+          </li>
+          <li>
+            TRANSACTIONS
+          </li>
+          <li>
+            <i class="fa fa-search"></i>
+          </li>
+        </ul>
+      </nav>
+  </header>
+
```

Comment:
[OPTIONAL] On the home page, it would be nicer to add a log-out button so that the user is able to sign out from your app and also to improve the user experience.

<img width="210" alt="image" src="https://user-images.githubusercontent.com/77190203/178049006-0b71520f-a828-4446-94da-25157a971ba1.png">